websocket.go

```diff
@@ -225,3 +238,62 @@ func (l *listener) Multiaddr() ma.Multiaddr {
 }
 
 var _ tpt.Transport = (*WebsocketTransport)(nil)
+
+type GorillaNetConn struct {
+	Inner              *wsGorilla.Conn
+	DefaultMessageType int
+}
+
+func (c GorillaNetConn) Read(b []byte) (n int, err error) {
```

Comment:
I got something, something not satisfying interface because of pointer receiver when I did that